astro-version/src/layouts/Settings.astro

```diff
@@ -67,7 +67,7 @@ const messages = settingsMessages(Astro.currentLocale)
 <style>
   .settings {
     display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(360px, 1fr));
+    grid-template-columns: repeat(auto-fill, var(--card-width));
```

Comment:
These are different cards, but since we apply the same styles it should work, but I agree to add separate variable here.